benchexec/tools/sesl.py

```diff
@@ -0,0 +1,54 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    A Symbolic Executor based on Separation Logic
+    https://spencerl-y.github.io/SESL/
+    """
+
+    REQUIRED_PATHS = [
+        "bin",
+        "libs",
+        "scripts",
+        "libs/slah/lib",
+        "libs/z3/lib",
+        "sesl-svcomp.sh",
```

Comment:
```suggestion
```
These lines are actually redundant, because you have `libs` in there anyway. And the main executable itself is automatically added.